EthOn.rdf

```diff
@@ -2220,7 +2149,8 @@
             </owl:Restriction>
         </rdfs:subClassOf>
         <suggestedStringRepresentation>Tx</suggestedStringRepresentation>
-        <rdfs:comment xml:lang="en">Transactions are Messages between two Accounts that may transfer Ether and may contain a payload. Transactions always originate from an External Account that is controlled by an External Actor by means of a private key.  The execution of a Transaction creates a &apos;Transaction Receipt&apos;.</rdfs:comment>
+        <rdfs:comment xml:lang="en">Transactions are Messages between two Accounts that may transfer Ether and may contain a payload. Transactions always originate from an External Account that is controlled by an External Actor by means of a private key.  The execution of a Transaction creates a TransactionReceipt.</rdfs:comment>
```

Comment:
Double space after `private key.` Doesn't seem to be displayed everywhere, it seems that the medium controls if double space is displayed or not (html ignores it sometimes) Should be fixed though.
> a private key.  The execution of